emails/2023-10-20.md

```diff
@@ -0,0 +1,25 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+For those of you that are new, the format of this email is a joke followed by (usually) three topics. The running joke is I hate writing the joke. This is one of those weeks where nothing is funny. Enjoy the update. 
+
+### Undrop
+
+Ever since Dolt supported `DROP DATABASE`, it bothered me that it was the only SQL query that was destructive by default in Dolt. One errant `DROP DATABASE` and you lose all your data and all of it's history. This made no sense. By contrast, a `DROP TABLE` could be undone with a simple `CALL DOLT_RESET()`.
```

Comment:
```suggestion
Ever since Dolt supported `DROP DATABASE`, it bothered me that it was the only SQL query that was destructive by default in Dolt. One errant `DROP DATABASE` and you lose all your data and all of its history. This made no sense. By contrast, a `DROP TABLE` could be undone with a simple `CALL DOLT_RESET()`.
```